static/src/javascripts/projects/common/modules/commercial/epic/iframe-epic-utils.js

```diff
@@ -0,0 +1,142 @@
+// @flow
+
+import {
+    insertAtSubmeta,
+    reportEpicError,
+} from 'common/modules/commercial/epic/epic-utils';
+import { getStyles } from 'commercial/modules/messenger/get-stylesheet';
+
+import type { EpicComponent } from 'common/modules/commercial/epic/epic-utils';
+
+export type IframeEpicComponent = EpicComponent & { iframe: HTMLIFrameElement };
+type FontName =
+    | 'GuardianHeadline'
+    | 'GuardianTextEgyptianWeb'
+    | 'GuardianTextSansWeb';
+
+// channel for messages between Optimize Epic and Guardian frontend
+const OPTIMIZE_EPIC_CHANNEL = 'OPTIMIZE_EPIC';
+
+// messages in this channel (incoming / outgoing) should have the following schema:
+// { channel: 'OPTIMIZE_EPIC', messageType: string, data: ?any }
+
+// incoming event types
+const EPIC_INITIALIZED = 'EPIC_INITIALIZED';
+const EPIC_HEIGHT = 'EPIC_HEIGHT';
+
+// outgoing event types
+const FONTS = 'FONTS';
+
+// Return type a Promise - presuming fastdom will be required (?)
+const createEpicIframe = (url: string): Promise<IframeEpicComponent> => {
```

Comment:
I'm thinking we don't actually need a Promise here? Since the element isn't actually inserted in the DOM yet so no need for fastdom

Reply:
And do we even need the containing `div`?